test/e2e/brand_customization_test.go

```diff
@@ -0,0 +1,179 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/testframework"
+)
+
+const (
+	testCustomProductName    = "custom product name"
+	customLogoVolumeName     = "custom-logo"
+	customBrandConfigMapName = "custom-brand-configmap"
+	customBrandImageKey      = "pic.png"
+	customLogoMountPath      = "/var/logo/"
+)
+
+// Test prep - setup the client used by each test
+func setupCustomTestCase(t *testing.T) (*testframework.Clientset, operatorsv1.ConsoleCustomization) {
+	client := testframework.MustNewClientset(t, nil)
```

Comment:
maybe add `testframework.MustManageConsole(t, client)` ?

Reply:
Added in my follow, thx!